### ISSUE TITLE
Travis CI no longer working

### ISSUE THREAD

**Issue by psibre** [marytts/gradle-marytts-voicebuilding-plugin#138]
https://travis-ci.org/ has [shut down](https://blog.travis-ci.com/2021-05-07-orgshutdown), and users need to migrate to https://travis-ci.com/.

The migration had already happened for many of our repositories (including this one -- see #118), but unfortunately, there is a fixed, per-org allowance of free build time, which was consumed *without warning* by routine CI pipeline runs. This somehow dipped the balance into the negative, and it seems to have stayed there.

Adding insult to injury, the ["Consume paid credits for OSS" setting](https://travis-ci.com/organizations/marytts/plan?anchor=oss-consumption) was not enabled (even though that should have been the default!) -- and the links to the plan pages are currently broken!

### TL;DR

We have no choice but to drop Travis CI and migrate to something else -- by default, GitHub Actions.
marytts/voice-cmu-awb-time#6 has already successfully done this.
